### ISSUE TITLE
Missed new version message

### ISSUE THREAD

**Issue by BeckyDTP** [dougmassay/tagmechanic-sigil-plugin#3]
Checking for a new plugin version works but the user will never be notified of the new version. The date of the check and the new version number are saved in json file, but the corresponding message is missing.

After installing the old version (0.4.5), the following message appears:
![tagmechanic-plugin_update](https://user-images.githubusercontent.com/22416021/97562993-bed70d00-19e2-11eb-8d40-2810184031cc.png)

In the new version (0.5.x) it is missing.

IMHO it's a pity that the message about the new version appears only once and is easy to miss.
If this message were to be given directly in a dialog box, it would not bother anyone:
![tagmechanic-update-label](https://user-images.githubusercontent.com/22416021/97563020-ca2a3880-19e2-11eb-8e0f-993b7c4917c9.png)

```
        if self.update:
            update_layout = QHBoxLayout()
            layout.addLayout(update_layout)
            self.label = QLabel()
            font = QFont()
            font.setPointSize(10)
            self.label.setFont(font)
            self.label.setText(_t('guiMain', 'Plugin Update Available'))
            self.label.setStyleSheet('QLabel {color: #5CC0FF;}')
            update_layout.addWidget(self.label)
```

What do you think?

**Comment by dougmassay** [comment]
Looks like I missed presenting the "update available" message entirely when moving to PyQt5! Eek! Thanks for catching it.

I'm not opposed to having it presented as you describe, but I rather prefer the "notify only once per new version" logic. I don't want to nag, and I don't want users who may consciously choose to stay with an older version to be constantly reminded that an update is available. If I go with your suggestion, I'd need a preference-based option for the user to turn off update notifications altogether.

I'm not crazy about the hard-coded font size change, either. It ignores the user's font-size choice from Sigil's preferences (and/or Qt Platform themes on Linux). I know Qt makes it nearly impossible to specify a relative font-size change, so I'm not sure it's worth the trouble of trying to tweak the size of the label's font from the qApp default.

I also wouldn't mind seeing the font's color derived from the link color of the qApp's current QPalette. That way, it can be assured of dynamically  accommodating any and all linux Qt platform themes that may be being employed.

**Comment by dougmassay** [comment]
I'm happy to tackle implementing my above suggestions, but I also wouldn't be opposed at all to a pull request if you're looking for a small project. ;)

**Comment by dougmassay** [comment]
In the beginning of guiMain's setup_ui method:

`app = QApplication.instance()`
`p = app.palette()`
`link_color = p.color(p.Active, p.Link).name()`

Then when constructing the update label:

`label.setStyleSheet('QLabel {{color: {};}}'.format(link_color))`

The double braces are necessary for escaping _actual_ curly braces whenever using Python's built-in string formatting/substitution.

**Comment by BeckyDTP** [comment]
Fixed by #4
